bin/plugin/commands/performance.js

```diff
@@ -260,6 +261,22 @@ async function runPerformanceTests( branches, options ) {
 	);
 
 	log( '>> Starting the WordPress environment' );
+	if ( options.wpVersion ) {
+		// Patch the environment's .wp-env.json config to use the specified WP version:
+		//
+		//     {
+		//         "core": "https://wordpress.org/wordpress-$VERSION",
+		//         ...
+		//     }
+		//
+		// Only use the major version, i.e. 5.7.2 becomes 5.7.
+		const major = options.wpVersion.split( '.' ).slice( 0, 2 ).join( '.' );
+		const zipUrl = `https:\\/\\/wordpress.org\\/wordpress-${ major }.zip`;
+		log( `Using WordPress version ${ major }` );
+		await runShellScript(
+			`sed -I '' -e '/"core":/s/WordPress\\/WordPress/${ zipUrl }/' "${ environmentDirectory }/.wp-env.json"`
+		);
```

Comment:
We could use `jq` -- or just plain JS -- to modify the JSON file here 😬

Reply:
> or just plain JS

True, true. 😅 How about now?